CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
    * Added missing unit tests for `pysat.utils.time`
    * Added missing unit tests for `pysat.utils.file.parse_delimited_filename`
    * Streamlined unit tests for `test_orbits`
+   * Updated class declaration consistent with python 3
```

Comment:
This is awkward and I don't know how to fix it 🐟

Reply:
So this issue is converting to the so-called "new style" class declaration.  Specifically `class ClassName():` to `class ClassName(object):`.

What about
```suggestion
   * Updated class declaration consistent with python 3 standards
```